.ci/build-n-run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+function build_example()
+{   
+    make -C examples || exit 1
+}
+
+function get_mod_list()
+{
+    #mod_list will contain all filename which doesn't appear in non-working without path and file extension
+    mod_list=$(echo `ls examples/*.ko | xargs -n 1 basename | sed 's/\.ko$//' && cat .ci/non-working` | tr ' ' '\n' | sort | uniq -u)
+}
+
+function run_mod() #test module 2 times
+{
+    ( sudo insmod "examples/$1.ko" && sudo rmmod "$1" ) || exit 1;
+    ( sudo insmod "examples/$1.ko" && sudo rmmod "$1" ) || exit 1;
+}
+
+function run_examples()
+{
+    for module in $mod_list; do
```

Comment:
Rewrite as following:
```bash
for module in $(get_mod_list); do
    run_mod "$module"
done
```

Reply:
You can rename `get_mod_list` to `list_mod` for consistency.

Reply:
It rasing error when I add '('  before `list_mod`
```bash
.ci/build-n-run.sh: line 24: list_mod: command not found
```
My code:
```bash
function run_examples()
{
    get_mod_list
    for module in $(list_mod); do
        echo "$module"
        run_mod "$module"
    done
}
```

Reply:
You can call the function in bash. See https://ryanstutorials.net/bash-scripting-tutorial/bash-functions.php
You have to dump something in function `list_mod` though.

Reply:
Cool, I don't know that before.
Thanks for letting me know that.